pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -39,6 +39,13 @@ servingInfo:
   bindAddress: https://0.0.0.0:8443
   certFile: /var/serving-cert/tls.crt
   keyFile: /var/serving-cert/tls.key
+`
```

Comment:
`exampleConfigMapData` vs `exampleManagedConfigData`, just to clarify its `managed`.